hbase-handler/src/test/results/negative/hbase_ddl.q.out

```diff
@@ -26,4 +26,4 @@ key                 	int                 	It is a column key
 value               	string              	It is the column string value
 	 	 
 #### A masked pattern was here ####
-FAILED: SemanticException [Error 10134]: ALTER TABLE can only be used for [ADDPROPS, DROPPROPS, ADDCOLS, REPLACE_COLUMNS, SETPARTITIONSPEC] to a non-native table  hbase_table_1
+FAILED: SemanticException [Error 10134]: ALTER TABLE can only be used for [ADDPROPS, DROPPROPS, ADDCOLS] to a non-native table  hbase_table_1
```

Comment:
What happens with HBase tables if we try replacing columns and setting partition spec?

Reply:
They get this SemanticException:
```
ALTER TABLE can only be used for [ADDPROPS, DROPPROPS, ADDCOLS] to a non-native table  hbase_table_1
```
HBase (and all other storage handlers except for Iceberg at the moment), should get this exception for alter commands other than SET/UNSET TBLPROPERTIES and ADD COLUMNS.

Reply:
Previously, when we were working on adding new alter commands for Iceberg, we kept adding these new operation types (rename columns, etc.) to the allowed list. However, there was only one global allowed list for all storage handler types. Now, the allowed list has been moved into the storage handler, so I've reverted the global list to its original form (before all our Iceberg changes started flowing in)